flink-formats/flink-avro/src/test/java/org/apache/flink/table/runtime/batch/AvroTypesITCase.java

```diff
@@ -58,7 +55,7 @@
 @RunWith(Parameterized.class)
 public class AvroTypesITCase extends TableProgramsClusterTestBase {
 
-	private static final User USER_1 = User.newBuilder()
+	private static final SimpleUser USER_1 = SimpleUser.newBuilder()
```

Comment:
Had to drop the logical types, because the `BatchTableEnvironment` fails if we have a `java.time.Instant` in the input DataSet. We extract `BasicTypeInfo.INSTANT_TYPE_INFO` for it, which is translated to `TIMESTAMP_WITH_LOCAL_TIME_ZONE` which fails in legacy planner.

Reply:
Thanks for the explanation.

Reply:
But it would still be nice to test all types for Avro but maybe to the new Blink planner. Could we add least have a separate commit for this class? Then we can revert the changes because after FLIP-136 this test should definitely work again, no? Or does it work already today?